packages/api/src/tasks/checkAlerts.ts

```diff
@@ -388,6 +388,13 @@ export default class CheckAlertTask implements HdxTask<CheckAlertsTaskArgs> {
       alertCount: alerts.length,
     });
 
+    if (!conn.password) {
+      const className = this.provider.constructor.name;
+      throw new Error(
+        `provider ${className} did not fetch password for connection ${conn.id}`,
+      );
```

Comment:
we can't throw here. its okay to add a warn logger

Reply:
Should it even be a `warn`? We know that in cloud we'll have passwordless connections and probably don't want to spam the log stream. I was thinking `info`.